### ISSUE TITLE
Cannot past link through cb_thunderlink button in Thunderbird 78.7.0

### ISSUE THREAD

**Issue by plesager** [CamielBouchier/cb_thunderlink#27]
Currently running Thunderbird 78.7.0 (64-bit) on Fedora 33., I am able to create links, which is wonderful, but I cannot open them. I tried to use the command line:

`<path-to>/cb_thunderlink/src/cb_thunderlink.py cbthunderlink://<message-id>` 

which first required converting `cb_thunderlink.py` to a Linux format (`dos2unix filename` did it) to execute.  Realizing that `register` also goes through that script I register again and reinstall the add-on. Clicking on the `cb_thunderlink` button I cannot paste the cbthunderlink - nothing happens. I expect a pop up window of some sort, right? Here is the `cb_thunderlink.log` (slightly edited to shorten paths and message-id):

> 2021-02-24 14:53:18,191 - 31007 - INFO -   83 : ==========================================================================================
2021-02-24 14:53:18,191 - 31007 - INFO -   84 : Starting cb_thunderlink (['..../src/cb_thunderlink.py', 'register'])
2021-02-24 14:53:18,262 - 31007 - INFO -  253 : Finishing cb_thunderlink
2021-02-24 14:53:18,263 - 31007 - INFO -  254 : ==========================================================================================
2021-02-24 14:54:39,584 - 31358 - INFO -   83 : ==========================================================================================
2021-02-24 14:54:39,584 - 31358 - INFO -   84 : Starting cb_thunderlink (['.../src/cb_thunderlink.py', '/home/sager/.mozilla/native-messaging-hosts/cb_thunderlink.json', 'cb_thunderlink@bouchier.be'])
2021-02-24 14:54:39,584 - 31358 - INFO -  224 : Interfacing with background.js
2021-02-24 14:54:39,584 - 31358 - INFO -  229 : Trying to bind to ('127.0.0.1', 1302)
2021-02-24 14:54:39,584 - 31358 - INFO -  235 : Bound to ('127.0.0.1', 1302)
2021-02-24 14:54:39,584 - 31358 - INFO -  237 : Start listening on ('127.0.0.1', 1302)
2021-02-24 14:58:24,196 - 31556 - INFO -   83 : ==========================================================================================
2021-02-24 14:58:24,196 - 31556 - INFO -   84 : Starting cb_thunderlink (['.../src/cb_thunderlink.py', 'cbthunderlink://MjAyMS0wMi0yM1QwOTo1NzowOS4wMDBaOyJQZX...'])
2021-02-24 14:58:24,196 - 31556 - INFO -  207 : Executing cb_thunderlink
2021-02-24 14:58:24,197 - 31556 - INFO -  211 : Message sent: cbthunderlink://MjAyM....XRlcnNAd3VyLm5sPg==
2021-02-24 14:58:24,197 - 31556 - INFO -  253 : Finishing cb_thunderlink
2021-02-24 14:58:24,197 - 31358 - INFO -  240 : Connection from ('127.0.0.1', 49846)
2021-02-24 14:58:24,197 - 31556 - INFO -  254 : ===========================================================================================

There doesn't seem to be any problem, no exceptions were thrown out. My interpretation is that the problem is with the add-on itself and not the `cb_thunderlink.py`. Is that correct? Any pointer for where I could look to fix the button issue?

**Comment by plesager** [comment]
Maybe related to #24. Going to check.

**Comment by plesager** [comment]
I went through the checklist on the blog. Turns out that global indexing was off. Now I have clickable links in org-mode working!
I've also discovered how the `cb_thunderlink` button works: If the last thing you copied is a valid `cb_thunderlink`, then clicking the button will lead you to the linked email. Somehow I missed that, maybe because I keep a history of copied items and switch between them.
I noticed that moving the email to a new folder, the link does not work anymore, even if I put back the email to its original folder. Maybe something for another issue or feature request. Anyway, thanks for making **cb_thunderlink** available!
